source/administration/production-notes.txt

```diff
@@ -740,20 +777,36 @@ accommodate Windows' commitment rules during peak database use.
 MongoDB on Virtual Environments
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-This section describes considerations when running MongoDB in some of the
-more common virtual environments.
+This section describes considerations when running MongoDB in some of 
+the more common virtual environments.
 
 For all platforms, consider :ref:`virtualized-disks-scheduling`.
 
-EC2
-```
+:abbr:`AWS (Amazon Web Services)` :abbr:`EC2 (Elastic Compute Cloud)`
+`````````````````````````````````````````````````````````````````````
+
+- When available, enable :abbr:`AWS (Amazon Web Services)`
+  `Enhanced Networking <http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/enhanced-networking.html#enabling_enhanced_networking>`_ 
+  for your instance. Not all instance types support Enhanced Networking. 
+
+  To learn more about Enhanced Networking, see to the 
+  `AWS documentation <http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/enhanced-networking.html#enabling_enhanced_networking>`_.
+
+- Use provisioned IOPS for the storage, with separate devices for
```

Comment:
As long as you're using the :abbr: directive for other abbreviations around here, might as well add it for IOPS too.